pkg/kstatus/polling/statusreaders/common.go

```diff
@@ -137,8 +137,8 @@ func toSelector(resource *unstructured.Unstructured, path ...string) (labels.Sel
 	return metav1.LabelSelectorAsSelector(&s)
 }
 
-func toIdentifier(u *unstructured.Unstructured) wait.ResourceIdentifier {
-	return wait.ResourceIdentifier{
+func toIdentifier(u *unstructured.Unstructured) object.ObjMetadata {
```

Comment:
Should this function go into the new object package?

Reply:
Yeah, I agree maybe we should move it. If you don't mind, I'll look at that in a separate PR.